### DISCUSSION TITLE
Pull request creation failed despite branch deletion, repository recreation, and account logout/login

### DISCUSSION THREAD

**Discussion by shyakadavis** [community/community#55772] [Pull Requests]
### Select Topic Area

Bug

### Body

Hello,

I'm encountering a persistent issue while attempting to create a pull request on this [repo](https://github.com/shyakadavis/flashcards). I have taken several steps to troubleshoot the problem, but the error persists.

Here are the actions I have already taken:

- Deleted the branch I was working on and created a separate branch.
- Deleted the repository entirely and re-created it.
- Logged out of my GitHub account on the browser and logged back in.

Despite these efforts, I continue to receive the following error message when trying to create a pull request:

`"Pull request creation failed. Validation failed: was submitted too quickly"`

<img width="1478" alt="Screenshot 2023-05-17 at 21 46 49" src="https://github.com/community/community/assets/87414827/92631fc9-4dcf-458d-98b8-633969229faa">

Further context

I think the problem initially arose from within Visual Studio Code (Version: 1.78.2 Universal).
I attempted to create the pull request from within the IDE using the "GitHub Pull Requests and Issues" extension (v0.64.0).
Unfortunately, the extension/IDE froze, so I decided to force-close Visual Studio Code.
I then decided to create the pull request directly from the browser, but that's when the issue first arose.

I would greatly appreciate any insights, guidance, or troubleshooting steps to resolve this issue.

Thank you for your assistance.

**Comment by shyakadavis** [comment]
For anyone who might face a similar issue in the future, I waited for an hour or so, and tried again, and it worked.

**Comment by metachris** [comment]
Still seeing this issue on several repositories

**Comment by DCPDamon** [comment]
Same thing here. I have the same situation. I tried a pull request in VS Code, it failed (auth issue), and now I'm trying to do the pull request in gituhib.com website and I'm getting the same error.  and it's not just pull requests. I can't even post on this issue from that account. I had to log into a different account just so I could comment here.

**Comment by shyakadavis** [comment]
@metachris @DCPDamon should I re-open this?

**Comment by DCPDamon** [comment]
@shyakadavis after an hour or so I tried again and it worked. No need to open it on my account.

**Comment by metachris** [comment]
Seems resolved for me too

**Reply by RoshdiRaed** [reply]
great

**Comment by bitbeckers** [comment]
Currently facing the same issue, so it's not resolved yet. Just waiting is the solution?

**Comment by raulmarindev** [comment]
I'm facing this issue too. Is it a temporary and random GitHub issue? Maybe related to this? https://github.com/microsoft/vscode-pull-request-github/issues/4813 I do have the GitHub pull requests extension installed.

**Comment by nurullahturkoglu** [comment]
I did get same issue today. Probably it is caused by the pull request extension on vscode

**Reply by RoshdiRaed** [reply]
Probably !!

**Comment by muddi900** [comment]
If the cause is the extension, then why does it persist on github.com?

**Comment by Ali-Hussein-dev** [comment]
same issue!
